webapp/src/views/EventList.vue

```diff
@@ -20,14 +20,15 @@
         </ul>
         <router-link
           :to="{ name: 'Register', params: { id: event.id } }"
-          class="btn btn-primary"
+          class="btn btn-outline-primary mt-3 w-100"
```

Comment:
Making this button outlined looks subtle but users might overlook the button easier than before. In this case that is especially problematic because most people want to register multiple persons after one another.

An alternative that might look good in combination with dark blue would be dark orange button.

Reply:
indeed,
I use a secondary btn now. Do you think that looks better?

Reply:
Not really convinced yet. Now it looks grayed out as if the button was disabled. We need to find a good secondary color or rather accent color to emphasize important UI elements.